src/aoc2020/day1.clj

```diff
@@ -0,0 +1,143 @@
+(ns aoc2020.day1
+  (:require [aoc2020.util :refer [file->seq]]))
+
+
+;; part 1
+
+(defn strs->ints [strs]
+  "parsing strings to integers"
+  (let [str->int (fn [str]
+                   (Integer/parseInt str))]
+    (map str->int strs)))
+
+(defn int->compl [n sum]
+  "calculate complement"
+  (- sum n))
+
+(defn compl-in-set? [n sum s]
+  "detect whether the complement is in the sequence (using set)"
+  (let [compl (int->compl n sum)]                           ; calculate complement
+    (contains? s compl)))                                   ; check if exist in set
+
+(defn int->int*compl [n sum]
```

Comment:
The naming does not reflect the idea presented in the domain.

Reply:
how about the name `product-of-compl`